Bottom: 36924a6196b3b963488424da44d51e34e8178d73
Top:    0a5044a2dd04b4163c6cfd68eb538adb4daede9c
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-17 15:32:30 -0500

Use original buildbot entrypoint and init daemon


---
diff --git a/docker/buildbot-master/Dockerfile b/docker/buildbot-master/Dockerfile
index b13cd1d..fcced3c 100644
--- a/docker/buildbot-master/Dockerfile
+++ b/docker/buildbot-master/Dockerfile
@@ -9,7 +9,6 @@ ARG BUILDBOT_TIMEZONE="Etc/UTC"
 
 RUN apk --no-cache add \
         openssh \
-        tini \
 &&  rm /etc/localtime \
 &&  ln -sf /usr/share/zoneinfo/${BUILDBOT_TIMEZONE} /etc/localtime \
 &&  echo "${BUILDBOT_TIMEZONE}" > /etc/timezone
@@ -27,5 +26,4 @@ RUN adduser -h "/home/buildbot" -s "/bin/sh" -D -u ${BUILDBOT_UID} buildbot \
 &&  chown -v -R buildbot:buildbot "/home/buildbot"
 
 USER buildbot
-ENTRYPOINT ["/sbin/tini", "--"]
 CMD ["/home/buildbot/start.sh"]
diff --git a/docker/buildbot-master/buildbot/buildbot.tac b/docker/buildbot-master/buildbot/buildbot.tac
index b87ce04..c2fac83 100644
--- a/docker/buildbot-master/buildbot/buildbot.tac
+++ b/docker/buildbot-master/buildbot/buildbot.tac
@@ -13,7 +13,7 @@ configfile = 'master.cfg'
 # note: this line is matched against to check that this is a buildmaster
 # directory; do not edit it.
 application = service.Application('buildmaster')
-# application.setComponent(ILogObserver, FileLogObserver(sys.stdout).emit)
+application.setComponent(ILogObserver, FileLogObserver(sys.stdout).emit)
 
 m = BuildMaster(basedir, configfile, umask=None)
 m.setServiceParent(application)
diff --git a/docker/buildbot-master/buildbot/start.sh b/docker/buildbot-master/buildbot/start.sh
index 5e0eed7..71002f8 100755
--- a/docker/buildbot-master/buildbot/start.sh
+++ b/docker/buildbot-master/buildbot/start.sh
@@ -8,5 +8,5 @@ chmod -R 600 ${HOME}/.ssh/*
 
 ln -v -s -f ${HOME}/buildbot.tac ${HOME}/master.cfg ${HOME}/*.py $BUILDBOT_DATA
 
-/usr/bin/buildbot --verbose upgrade-master
-exec twistd --nodaemon --logfile=- --pidfile=/tmp/twistd.pid --python=buildbot.tac
+# Run original buildbot entrypoint
+exec "/usr/src/buildbot/docker/start_buildbot.sh"
